mlflow/data.py

```diff
@@ -68,7 +68,10 @@ def parse_gs_uri(uri):
 
 def is_uri(string):
     parsed_uri = urllib.parse.urlparse(string)
-    return len(parsed_uri.scheme) > 0
+    if os.name == 'nt':
+        return not re.match('^[a-zA-Z]:\\\\.*', string)
```

Comment:
```suggestion
        return not re.match(r'^[a-zA-Z]:\\.*', string)
```
You can use raw string here.

Reply:
And can we put this in a function (named like `is_windows_drive_path`) and write a test?

Reply:
@harupy, when I tried raw string with two slashes the function errored out with “Expected string or bytes-like object”.
Function has been moved, will write test.

Reply:
Looks like you passed `None` to `string`. Raw string is not the cause of the error.

![Screen Shot 2020-08-07 at 2 59 23](https://user-images.githubusercontent.com/17039389/89565782-06258580-d85a-11ea-9799-8615ca61eefd.png)

https://github.com/mlflow/mlflow/runs/951541560#step:5:162

Reply:
Good catch, thanks!